lib/config-loader.js

```diff
@@ -119,6 +119,8 @@ module.exports = () => {
   // 2.c. needed files
   if (!fs.existsSync(config.manifest.src) || !fs.statSync(config.manifest.src).isFile()) throw new Error('Missing manifest.yml file')
   if (!fs.existsSync(_abs('package.json')) || !fs.statSync(_abs('package.json')).isFile()) throw new Error('Missing package.json file')
+  // 2.d missing cna hostname
+  if (config.app.hasFrontend && !userConfig.cna.hostname) throw new Error('Missing cna.hostname .aio config (or AIO_CNA_HOSTNAME env)')
```

Comment:
as @purplecabbage mentioned would be good to have a default value to make it easier for users to get started